packages/edit-site/src/components/page-patterns/patterns-list.js

```diff
@@ -60,7 +60,7 @@ export default function PatternsList( { categoryId, type } ) {
 
 	const { patterns, isResolving } = usePatterns(
 		type,
-		categoryId !== 'uncategorized' ? categoryId : '',
+		type !== PATTERNS || categoryId !== 'uncategorized' ? categoryId : '',
```

Comment:
The following avoids the `||` and ternary combination and seems slightly easier to understand what is happening and seems to work in my quick testing:
```suggestion
		categoryId === 'uncategorized' && type === PATTERNS ? '' : categoryId,
```
what do you think?

Reply:
I think it might be time to also extract this to its own variable for readability's sake.